src/components/pages/community-champions/meet-the-champions.js

```diff
@@ -0,0 +1,180 @@
+import LocationPinIcon from '~images/community-champions/location-pin-grey.svg';
+import ChampionPlaceholderImage from '~images/community-champions/champion-placeholder.svg';
+
+const CHAMPION_CONTAINER_WIDTH = '200px';
+const CHAMPION_CONTAINER_MOBILE_WIDTH = '170px';
+const PROFILE_PICTURE_SIZE = 112;
+const PROFILE_PICTURE_GRADIENT_OFFSET = 8;
+const LOCATION_CONTAINER_TOP_MARGIN = '4px';
+const LOCATION_CONTAINER_MOBILE_TOP_MARGIN = '2px';
+const PIN_LOCATION_IMAGE_RIGHT_MARGIN = '4px';
+const CHAMPION_MOBILE_VERTICAL_MARGIN = '4px';
+
+const Title = styled(H1)`
+    margin-bottom: calc(${size.xlarge} - ${size.xsmall});
+    text-align: center;
+    @media ${screenSize.upToLarge} {
+        margin-bottom: ${size.medium};
+    }
+`;
+
+const Container = styled('div')`
+    background-color: ${({ theme }) => theme.colorMap.greyDarkThree};
+    margin: ${size.xlarge} -${size.xxlarge};
+    padding: ${size.xlarge} ${size.xxlarge}
+        calc(${size.xlarge} - ${size.xsmall});
+    @media ${screenSize.upToLarge} {
+        margin: ${size.large} -${size.default};
+        padding: ${size.large} ${size.default}
+            calc(${size.large} - ${CHAMPION_MOBILE_VERTICAL_MARGIN});
+    }
+`;
+
+const ChampionsContainer = styled('div')`
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+`;
+
+const ChampionLink = styled(Link)`
+    align-items: center;
+    border-radius: ${size.xsmall};
+    display: flex;
+    flex-direction: column;
+    margin: ${size.xsmall} 0;
+    padding: ${size.default};
+    text-align: center;
+    text-decoration: none;
+    width: ${CHAMPION_CONTAINER_WIDTH};
+    @media ${screenSize.upToLarge} {
+        margin: ${CHAMPION_MOBILE_VERTICAL_MARGIN} 0;
+    }
+    @media ${screenSize.upToMedium} {
+        padding: ${size.default} 0;
+        width: ${CHAMPION_CONTAINER_MOBILE_WIDTH};
+    }
+    &:hover,
+    &:focus {
+        background-color: ${({ theme }) => theme.colorMap.greyDarkTwo};
+        color: inherit;
+        cursor: pointer;
+    }
+`;
+
+const LocationPinImage = styled('img')`
+    margin-right: ${PIN_LOCATION_IMAGE_RIGHT_MARGIN};
+    vertical-align: text-top;
+`;
+
+const LocationText = styled(P4)`
+    color: ${({ theme }) => theme.colorMap.greyLightTwo};
+    margin-top: ${LOCATION_CONTAINER_TOP_MARGIN};
+    @media ${screenSize.upToMedium} {
+        margin-top: ${LOCATION_CONTAINER_MOBILE_TOP_MARGIN};
+    }
+`;
+
+const Location = ({ location }) => (
+    <LocationText collapse>
+        <LocationPinImage src={LocationPinIcon} alt="Location pin" />
+        {location}
+    </LocationText>
+);
+
+const Occupation = styled(P3)`
+    color: ${({ theme }) => theme.colorMap.greyLightTwo};
+`;
+
+const ProfilePicture = styled(ProfileImage)`
+    margin-bottom: ${size.default};
+    div {
+        ${props =>
+            props.image
+                ? `background-color: ${props.theme.colorMap.devWhite};`
+                : ''}
```

Comment:
${({image}) => image ? `background-color: ${props.theme.colorMap.devWhite};` : ''};`

Might be a bit cleaner?

Reply:
🤩